pkg/apis/extensions/types.go

```diff
@@ -926,6 +926,7 @@ var (
 	Quobyte               FSType = "quobyte"
 	AzureDisk             FSType = "azureDisk"
 	PhotonPersistentDisk  FSType = "photonPersistentDisk"
+	StorageOS             FSType = "storageos"
```

Comment:
It looks like the list in [v1beta1/types.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/extensions/v1beta1/types.go#L940) has already diverged, please keep them consistent if possible.

Also, is it intentional to make these variables instead of constants?